modules/vaos/app/controllers/vaos/pact_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module VAOS
+  class PactController < ApplicationController
+    def index
+      binding.pry
```

Comment:
Looks like a leftover debugging line 🐛

Reply:
I think this has been resolved, but I'm not sure why it's still appearing in the diff.